zap/deployment.yaml

```diff
@@ -93,6 +93,11 @@ data:
               secretKeyRef:
                 name: ${JOB_SECRET}
                 key: DRIVE_ROOT_ID
+          - name: DRIVE_ID
```

Comment:
Maybe? It's being set as a secret in k8s, similar to the other variables being exported as secrets here. Are the other ones in the cloud build config? Where is that and what does it do?

Reply:
Tom, I think it works fine to set it as a secret, even if it is not a real secret. In case it is accidentally changed in the cloud build config, it can fail all the file uploads in the GDrive.